docs/whats-new.asciidoc

```diff
@@ -14,143 +14,164 @@ Other versions: {security-guide-all}/8.14/whats-new.html[8.14] | {security-guide
-image::whats-new/images/8.15/timeline-notes-flyout.png[Example Timeline with the notes flyout highlighted]
+image::whats-new/images/8.16/visualize-tab-lp-alert-details.gif[Examine alert details from event analyzer, 80%]
 
 [float]
-== Response actions enhancements
+=== Resize alert and event details flyouts
 
-[float]
-=== Scan files and folders for malware
+You can now resize the alert and event details flyouts and choose how they're displayed—over the Alerts table or next to it.
+
+[role="screenshot"]
+image::whats-new/images/8.16/flyout-settings.gif[Change alert details flyout settings]
 
-{elastic-defend}'s new {security-guide}/response-actions.html#_scan[`scan` response action] lets you perform on-demand malware scans of a specific file or directory on a host. Scans are based on the malware protection settings configured in your {elastic-defend} integration policy.
+[float]
+== {elastic-defend} and response actions enhancements
 
 [float]
-=== Isolate and release CrowdStrike-enrolled hosts
+=== More SentinelOne third-party response actions
 
-Using Elastic's CrowdStrike integration and connector, you can now perform {security-guide}/third-party-actions.html#crowdstrike-response-actions[response actions] on hosts enrolled in CrowdStrike's endpoint protection system. These actions are available in this release:
+Additional third-party response actions are available using Elastic's {security-guide}/third-party-actions.html#sentinelone-response-actions[SentinelOne] integration and connector:
 
-* Isolate a host from the network
-* Release an isolated host
+* Get processes
+* Terminate a process
 
 [float]
-=== Retrieve files from SentinelOne-enrolled hosts
+=== {elastic-defend}'s automated response actions support all rule types
+
+You can now configure any detection rule type to perform {elastic-defend}'s {security-guide}/automated-response-actions.html[automated response actions].
 
-Using Elastic's SentinelOne integration and connector, you can now {security-guide}/third-party-actions.html#sentinelone-response-actions[retrieve files] from SentinelOne-enrolled hosts and download them through {elastic-sec}.
+////
+Commenting out until docs are ready
 
 [float]
-== Filter out process descendants
+=== New rules for {elastic-defend}'s endpoint protection features
 
-Create an {security-guide}/event-filters.html[event filter] that excludes the descendant events of a specific process, but still includes the primary process itself. This can help you limit the amount of events ingested into {elastic-sec}.
+New prebuilt rules tailored for each of {elastic-defend}'s endpoint protection features—malware, ransomware, memory threats, and malicious behavior—allow you to configure actions tailored for detection or prevention of each type.
+////
 
 [role="screenshot"]
-image::whats-new/images/8.15/event-filter-process-descendants.png[Add event filter flyout, 70%]
+image::whats-new/images/8.16/endpoint-protection-rules.png[Endpoint protection rules]
```

Comment:
Need to comment out this image, too — it shows the new prebuilt rules for Defend.

Reply:
Good catch, thanks!